package/rubygem-aytests.changes

```diff
@@ -1,9 +1,15 @@
 -------------------------------------------------------------------
+Tue Jan 12 08:36:25 UTC 2016 - igonzalezsosa@suse.com
+
+- Return a proper error code after running the tests
+- Show a summary of passed, failing a missing tests
+- 1.0.3
+
+-------------------------------------------------------------------
 Mon Jan 11 12:55:49 UTC 2016 - mfilka@suse.com
 
 - fixed MAC formatting when passed as parameter to virtualbox
   during machine creation
-- 1.0.3
```

Comment:
Yes, it's intentional. We didn't release 1.0.3 yesterday (I was waiting to get this fix merged).